client/app/layouts/splash.html

```diff
@@ -21,7 +21,7 @@
   <div layout="column" layout-align="center center" flex="noshrink" layout-margin>
     <div layout layout-align="center center">
       <logo></logo>
-      <h1 class="title">Foodsaving Worldwide</h1>
+      <heading align="center">Foodsaving<br>Worldwide</heading>
```

Comment:
Thanks, that was a great article.  Tere are only two options- <h1-h6> or <header>. <header> might sound a tad ambiguous. I guess <h1> makes more sense here?